src/main/scala/org/broadinstitute/dsde/agora/server/business/AuthorizedAgoraEntity.scala

```diff
@@ -3,4 +3,4 @@ package org.broadinstitute.dsde.agora.server.business
 
 import org.broadinstitute.dsde.agora.server.model.AgoraEntity
 
-case class AuthorizedAgoraEntity(entity: AgoraEntity, authorization: AgoraPermissions)
+case class AuthorizedAgoraEntity(entity: Option[AgoraEntity], authorization: AgoraPermissions)
```

Comment:
Good question - I probably should have explained this. See QueryHandler line 50. When trying to get a single method by namespace/name/snapshotId, we get Some(Entity) or None in response. Then we have to check for permissions, which involves creating an AuthorizedAgoraEntity. I didn't want to have to figure out whether the response was Some or None before doing the auth check.